webpack/components/Table/TableWrapper.js

```diff
@@ -113,6 +119,7 @@ const TableWrapper = ({
       >
         {children}
       </MainTable>
+      <Flex><ShowPageControls variant={PaginationVariant.bottom} /></Flex>
```

Comment:
Prefer the child(ren) of `<Flex>` to be on a separate line.. just looks nicer to me